core/src/test/java/apoc/export/graphml/ExportGraphMLTest.java

```diff
@@ -210,6 +210,21 @@ public void testImportGraphMLWithoutCharactersDataKeys() throws Exception {
 
         TestUtil.testCall(db, "MATCH  ()-[c:RELATED]->() RETURN COUNT(c) AS c", null, (r) -> assertEquals(1L, r.get("c")));
     }
+    
+    @Test
+    public void testImportDefaultRelationship() throws Exception {
+        db.executeTransactionally("MATCH (n) DETACH DELETE n");
+        db.executeTransactionally("CALL apoc.import.graphml( " +
+                "  'http://sonetlab.fbk.eu/data/social_networks_of_wikipedia/vecwiki_social_network_extraction/vecwiki-20091230-manual-coding.graphml', " +
+                "  { batchSize: 5000, readLabels: true, defaultRelationshipType:'RELATED' } " +
+                ")");
+        TestUtil.testCall(db, "MATCH ()-[r]-() RETURN Distinct type(r) as type",
+                (r) -> {
+                    String label = (String) r.get("type");
+                    assertEquals(label, "RELATED");
```

Comment:
occhio per favore:
```suggestion
                    assertEquals("RELATED", label);
```

Il primo argomento è quello che ci aspettiamo sia corretto.
E' sempre così se ci sono altri posti dove lo hai fatto correggi

Reply:
Anche questo errore è dettato da pigrizia, quando leggi una api di solito i nomi degli argomenti sono *parlanti*